lib/StaticNarrative/exporter/processor_util.py

```diff
@@ -251,20 +251,35 @@ def get_data_icon(obj_type: str) -> dict[str, str]:
     return icon_info
 
 
-def get_authors(ws_client: Workspace, config: Config, wsid: str) -> list[dict[str, str]]:
+def get_display_names(auth_url: str, token: str, user_ids: list[str]) -> dict[str, Any]:
+    """Given a list of user IDs, get the corresponding display names."""
+    headers = {"Authorization": token}
+    r = requests.get(
+        f"{auth_url}/api/V2/users/?list=" + ",".join(user_ids),
+        headers=headers,
+        timeout=30 * 60,  # same as baseclient default
+    )
```

Comment:
Just use requests to access the auth API directly (this is how the narrative does it, too).